components/cdc/src/delegate.rs

```diff
@@ -621,6 +621,7 @@ impl Delegate {
         let locks = match &self.lock_tracker {
             LockTracker::Prepared { locks, .. } => locks,
             _ => {
+                advance.blocked_on_scan += 1;
```

Comment:
It's better to be `advance.blocked_on_scan += self.downstreams.len()`.

Reply:
Fixed.